lib/fastlane/actions/slather.rb

```diff
@@ -49,22 +49,23 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :build_directory,
                                        env_name: "FL_SLATHER_BUILD_DIRECTORY", # The name of the environment variable
                                        description: "The location of the build output", # a short description of this parameter
-                                       verify_block: proc do |value|
-                                         raise "No Build Directory specified, pass using `build_directory: 'location/of/your/build/output'`".red unless value and !value.empty?
-                                       end),
-          FastlaneCore::ConfigItem.new(key: :proj,
-                                       env_name: "FL_SLATHER_PROJ", # The name of the environment variable
+                                       optional: true
+                                      ),
+          FastlaneCore::ConfigItem.new(key: :xcodeproj,
```

Comment:
Sorry, we can't just change the key of this option, as it would break existing setups. Please revert this change to be `:proj` again.